src/Compilers/CSharp/Portable/Symbols/TypeWithAnnotations.cs

```diff
@@ -75,7 +75,7 @@ internal TypeWithAnnotations ToType()
         /// <summary>
         /// The underlying type, unless overridden by _extensions.
         /// </summary>
-        private readonly TypeSymbol _defaultType;
+        internal readonly TypeSymbol __defaultType;
```

Comment:
>__defaultType [](start = 37, length = 13)

Names of internal fields should start with a capital letter - ```DefaultType```

Reply:
Fixed